analysis_options.yaml

```diff
@@ -35,7 +35,6 @@ linter:
   - no_runtimeType_toString
   - package_api_docs
   - prefer_const_declarations
-  - prefer_expression_function_bodies
```

Comment:
Why? Is nice!

Reply:
I like the ability to decide whether to use them or not based on which gives the best readability. This lint complains all the time, even when the non-expression version reads better.